contracts/util/SettingsConsts.sol

```diff
@@ -78,4 +78,9 @@ contract SettingsConsts {
      */
     bytes32
         public constant OVER_COLLATERALIZED_BUFFER_SETTING = "OverCollateralizedBuffer";
+    /**
+        @notice The setting name for the maximum total value locked.
+        @notice The maximum total value locked is the supply cap for protocol, above which no additional deposits will be accepted.
```

Comment:
Suggestion:        @notice The maximum total value locked is the supply cap for a pool protocol, above which no additional deposits will be accepted on this pool.

Reply:
Added in